tests/chart_tests/test_authsidecar.py

```diff
@@ -187,6 +226,7 @@ def test_authSidecar_houston_configmap_with_annotation(self, kube_version):
             "tag": "my-custom-tag",
             "port": 8084,
             "pullPolicy": "IfNotPresent",
+            "resources": {"limits": {"cpu": "1000m", "memory": "1024Mi"}, "requests": {"cpu": "500m", "memory": "512Mi"}},
```

Comment:
this can also be defined as default_resources in the top of the class or function

Reply:
Let me put it at class level and use same variable everywhere.

Reply:
@pgvishnuram done